pisa/stages/osc/pi_prob3.py

```diff
@@ -186,14 +276,67 @@ def compute_function(self):
             self.data.link_containers('nubar', ['nuebar_cc', 'numubar_cc', 'nutaubar_cc',
                                                 'nuebar_nc', 'numubar_nc', 'nutaubar_nc'])
 
+        # this can be done in a more clever way (don't have to recalculate all paths)
+        YeI = self.params.YeI.value.m_as('dimensionless')
+        YeO = self.params.YeO.value.m_as('dimensionless')
+        YeM = self.params.YeM.value.m_as('dimensionless')
+        if YeI != self.YeI or YeO != self.YeO or YeM != self.YeM:
```

Comment:
cf. above

Reply:
Assuming the "if" is intended to do what it does, I understand this should stay the way it is, just like in your branch?

Reply:
Right